ads/google/a4a/utils.js

```diff
@@ -301,6 +301,8 @@ export function googlePageParameters(a4a, startTime) {
           'top': win != win.top ? topWindowUrlOrDomain(win) : null,
           'loc': win.location.href == canonicalUrl ? null : win.location.href,
           'ref': referrer || null,
+          'bdt': win.performance && win.performance.timing ?
```

Comment:
This is similar to ads/google/a4a/utils.js#getNavStart  How about we make a generic function there (getTiminingEvent) where name is passed?

Reply:
Done